Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -214,6 +236,10 @@
     Telecrystal: 50
   categories:
   - UplinkWeaponry
+  conditions:
+  - !type:BuyerSpeciesCondition
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)